src/edu/cmu/tartan/xml/XmlWriter.java

```diff
@@ -129,23 +130,30 @@ public static void overWriteFile(String destFileName, String sourceFileName) thr
         File dest = new File(destFileName);
 
         //copy file conventional way using Stream
-        InputStream is = null;
-        OutputStream os = null;
-        
-        is = new FileInputStream(source);
-        os = new FileOutputStream(dest);
-        byte[] buffer = new byte[1024];
-        int length;
-
-        while ((length = is.read(buffer)) > 0) {
-        	os.write(buffer, 0, length);
+        try (	InputStream is= new FileInputStream(source);
```

Comment:
Miss white space?
Unnecessary white space before InputStream.
Absent white space between is and '='.

Reply:
to match space with below statement